src/build/app/mod.rs

```diff
@@ -754,9 +758,6 @@ impl<'help> App<'help> {
     ///     .help_template("{bin} ({version}) - {usage}")
     /// # ;
     /// ```
-    /// [`App::about`]: ./struct.App.html#method.about
```

Comment:
I think these should exist and the links to them in the above description such that people will have an idea where there are coming from.

Reply:
Sure. I removed them because there's missing nuance; e.g. sometimes it's `long_about`. Do you think I should mention that or just keep it simple?

Reply:
Let's say about or long_about and link them both. No need to explain more.

Reply:
Made some edits in 2c91800.